src/components/add-ticket/hooks/use-housing-options.ts

```diff
@@ -1,6 +1,13 @@
 import { useTranslations } from "../../../hooks/translations"
 import dayjs from "dayjs"
 
+export const housingLabels = {
```

Comment:
this can be dropped now I think

Reply:
I need this to map to stuff :/

Will move the translations here :)